css/desktop.css

```diff
@@ -0,0 +1,337 @@
+    grid-template-columns: repeat(3, minmax(0, 1fr));
+    grid-gap: 1rem;
+    overflow-wrap: break-word;
+  }
+
+  #other-projects div {
+    padding-top: 11.125rem;
+  }
+
+  #project-1 {
+    background-image: url("../images/projects/project-1.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  #project-1 .project-heading,
+  #project-1 .project-description,
+  #project-1 .project-lang {
+    visibility: hidden;
+  }
+
+  #project-2,
+  #project-5 {
+    background-image: url("../images/projects/project-2.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  #project-3,
+  #project-6 {
+    background-image: url("../images/projects/project-3.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  #project-4 {
+    background-image: url("../images/projects/project-4.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  /* ABOUT ME */
+  #about-me h2 {
+    color: var(--mainColor);
+    font-size: 4.5rem;
+    font-weight: 400;
+    font-family: "Crete Round", sans-serif;
+  }
+
+  #about-me button {
+    margin: 2rem 0 8rem 0;
+  }
+
+  #about-me-div h2 {
+    margin-top: 5rem;
+  }
+
+  #about-me-div p {
+    font-size: 1.25rem;
+  }
+
+  #about-me-main-content {
+    flex-direction: row;
+  }
+
+  #about-me ul {
+    flex-direction: column;
+    align-items: normal;
+  }
+
+  #about-me-div {
+    width: 50%;
+    padding: 0 0 0 8.8125rem;
+  }
+
+  #about-me-empty-div {
+    background-image: url("../images/about-me/desktop-about-me.png");
+    background-position: right;
+    background-size: contain;
+    width: 50%;
+    height: auto;
+  }
+
+  #about-me-div hr {
+    width: 85%;
+  }
+
+  #about-me-lang {
+    display: flex;
+    padding: 0 8.875rem 10rem;
+    justify-content: space-between;
+  }
+
+  #about-me-lang div {
+    margin-top: 5rem;
+  }
+
+  #about-me-lang div ul {
+    align-items: flex-start;
+  }
+
+  #about-me-lang div ul li:hover,
+  #about-me-lang div div img:hover,
+  #about-me-lang div ul li:focus,
+  #about-me-lang div div img:focus {
+    transform: scale(0.95);
+  }
+
+  /* FORM */
+  #form {
+    display: flex;
+    gap: 2rem;
+    margin: auto;
+  }
+
+  #form h2 {
+    text-align: left;
+    font-size: 2.5rem;
+    width: 28.4375rem;
+    margin: 0 0 0 8.4375rem;
+    padding: 0;
+  }
+
+  #contact-form {
+    padding: 0;
+    width: 65%;
+    margin: 0 8.4375rem 0 0;
+  }
+
+  #form-button {
+    margin-left: 0;
+  }
+
+  #email,
+  #name {
+    width: 100%;
+  }
+
+  #message {
+    margin-bottom: 1.5rem;
+  }
+
+  /* CONTACT SOCIAL MEDIA */
+  #contact-social-form {
+    display: flex;
+    justify-content: center;
+    width: unset;
+  }
+
+  #social-media-footer-img {
+    display: none;
+  }
+
+  /* PROJECT DETAILS */
+  .section-menu {
+    left: 1rem;
+    right: 1rem;
+    top: 1rem;
+    bottom: 1rem;
+    display: flex;
+    flex-direction: column;
+    padding-top: 0.5rem;
```

Comment:
- [ ] [optional] Please kindly increase the padding a bit so as to have something very close to the design. Following design template to details shows that we can perfectly deliver whatsoever given to us as developers. Thanks. 😄

Reply:
Thanks for the review. I will do that.